File: 04-if-else-statement-and-loops/cpp/loop.cpp
#include<iostream>
using namespace std;

int main(){

    //for loop
    //ex=1 =>print coder kamlesh 5 times
    // for(int i=0;i<5;i=i+1){
    //     cout<<"Coder Kamlesh"<<endl;
    // }

    // ex=2 => print n natural of number
    // int n;
    // cout<<"Give number: ";
    // cin>>n;
    // for(int i=1;i<=n;i=i+1){
    //     cout<<i<<endl;
    // }

    // ex=3=> print square
    // int n;
    // cout<<"Enter no: ";
    // cin>>n;
    // for(int i=1;i<=n;i=i+1){
    //     cout<<i<<" Square is: "<<i*i<<endl;
    // }

    //ex=4 => Print all Even no upto n;
    int n;
    cout<<"Enter the no: ";
    cin>>n;
    for(int i=1;i<=n;i=i+1){
        if(i%2==0){
            cout<<i<<endl;
        }
    }
}

File: 04-if-else-statement-and-loops/java/loop.java
import java.util.Scanner;

public class  loop{
    public static void main(String[] args) {

        //for loop
        //ex=1 =>print coder kamlesh 5 times
        // for(int i=0;i<5;i++){
        //     System.out.println("hello coder kamlesh");
        // }

         // ex=2 => print n natural of number
         //make scanner
         Scanner scanner=new Scanner(System.in);
        //  System.out.println("Give no to print n natural of number: ");
        //  int n=scanner.nextInt();
        //  for(int i=1;i<=n;i=i+1){
        //     System.out.println(i);
        //  }

        //ex=3=>print square
        // System.out.print("Enter the no: ");
        // int n=scanner.nextInt();
        // for (int i = 1; i <= n; i = i + 1) {
        //     System.out.println(i + " Square is: " + (i * i));
        // }

        // ex=4=> print all even no upto n
        System.out.print("Enter the no: ");
        int n=scanner.nextInt();
        for (int i = 1; i <= n; i = i + 1) {
            if(i%2==0){
                System.out.println(i);
            }
        }
    }
}